app/src/newdocs/content/canvasModules/ARIMA-98.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable quotes, quote-props, indent, comma-dangle, max-len */
+import moduleDescription from './ARIMA-98.mdx'
+
+export default {
+  "id": 98,
+  "name": "ARIMA",
+  "path": "Time Series: Prediction",
+  "help": {
+    "outputNames": [
+      "pred"
+    ],
+    "inputs": {
+      "in": "Incoming time series"
+    },
+    "helpText": moduleDescription,
+    "inputNames": [
+      "in"
+    ],
+    "params": {},
+    "outputs": {
+      "pred": "ARIMA prediction"
+    },
+    "paramNames": []
+  }
+}
```

Comment:
Maybe instead of disabling these eslint rules, you could use `eslint --fix` on all these files and it'd just correct them automatically for you?

e.g.

```
npx eslint --fix app/src/newdocs/content/canvasModules/*.jsx
```

Reply:
almost works! seems to be unable to fix the max-length of a line but it's an improvement 👍